content/en/docs/reference/glossary/cgroups.md

```diff
@@ -0,0 +1,13 @@
+---
+title: CGroups
+id: cgroups
```

Comment:
```suggestion
id: cgroup
```

(because most glossary entries are singular). It would be OK to change the text as well to describe what a single cgroup is.